src/commands/cloud-run/flare.ts

```diff
@@ -0,0 +1,243 @@
+   */
+  public async execute() {
+    this.context.stdout.write(helpersRenderer.renderFlareHeader('Cloud Run', this.isDryRun))
+
+    const errorMessages: string[] = []
+    // Validate service
+    if (this.service === undefined) {
+      errorMessages.push(helpersRenderer.renderError('No service specified. [-s,--service]'))
+    }
+
+    // Validate project
+    if (this.project === undefined) {
+      errorMessages.push(helpersRenderer.renderError('No project specified. [-p,--project]'))
+    }
+
+    // Validate region
+    if (this.region === undefined) {
+      errorMessages.push(helpersRenderer.renderError('No region specified. [-r,--region]'))
+    }
+
+    // Validate Datadog API key
+    this.apiKey = process.env[CI_API_KEY_ENV_VAR] ?? process.env[API_KEY_ENV_VAR]
+    if (this.apiKey === undefined) {
+      errorMessages.push(
+        helpersRenderer.renderError(
+          'No Datadog API key specified. Set an API key with the DATADOG_API_KEY environment variable.'
+        )
+      )
+    }
+
+    // Validate case ID
+    if (this.caseId === undefined) {
+      errorMessages.push(helpersRenderer.renderError('No case ID specified. [-c,--case-id]'))
+    }
+
+    // Validate email
+    if (this.email === undefined) {
+      errorMessages.push(helpersRenderer.renderError('No email specified. [-e,--email]'))
+    }
+
+    // If there are errors, print them and exit
+    if (errorMessages.length > 0) {
+      for (const message of errorMessages) {
+        this.context.stderr.write(message)
+      }
+
+      return 1
+    }
+
+    // Get GCP credentials
+    this.context.stdout.write(chalk.bold('\n🔑 Verifying GCP credentials...\n'))
+    if (!(await checkAuthentication())) {
+      this.context.stderr.write(renderAuthenticationInstructions())
+
+      return 1
+    }
+    this.context.stdout.write('GCP credentials verified!\n')
+
+    // Get and print service configuration
+    this.context.stdout.write(chalk.bold('\n🔍 Fetching service configuration...\n'))
+    const runClient = new ServicesClient()
+    let config: IService
+    try {
+      config = await getCloudRunServiceConfig(runClient, this.service!, this.project!, this.region!)
+    } catch (err) {
+      if (err instanceof Error) {
+        this.context.stderr.write(helpersRenderer.renderError(`Unable to fetch service configuration: ${err.message}`))
+      }
+
+      return 1
+    }
+    maskConfig(config)
+    const configStr = util.inspect(config, false, 10, true)
+    this.context.stdout.write(`\n${configStr}\n`)
+
+    // Save and zip service configuration
+    this.context.stdout.write(chalk.bold('\n💾 Saving configuration...\n'))
+    const rootFolderPath = path.join(process.cwd(), FLARE_OUTPUT_DIRECTORY)
+    try {
+      // Delete folder if it already exists
+      if (fs.existsSync(rootFolderPath)) {
+        deleteFolder(rootFolderPath)
+      }
+
+      // Create folder
+      createDirectories(rootFolderPath, [])
```

Comment:
As of now, we pass in an empty array because no subfolders are created. Future PRs will create subfolders.